README.md

```diff
@@ -9,14 +9,17 @@ A Clojure/ClojureScript library of statistical sampling and transducing function
 * Beta-binomial
 * Binomial
 * Categorical
+* Cauchy
 * Chi-Squared
 * Dirichlet
 * Dirichlet-multinomial
 * Exponential
 * F
 * Gamma
+* Lognormal
```

Comment:
You guessed it, I'd prefer Log-normal here.

Reply:
Ok. Will change.